### PR TITLE
Update/cherry pick peekjob

### PR DESCRIPTION
**Pull request by ens-bwalts**
## Requirements

- Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion;
- Review the [development guidelines](https://ensembl-hive.readthedocs.io/en/master/dev/development_guidelines.html#ehive-development-in-a-nutshell) for eHive; remember in particular:
    - Do not modify code without testing for regression.
    - Provide simple unit tests to test the changes.
    - If you change the database schema, please follow the instructions [for schema changes in the developer guidelines](https://ensembl-hive.readthedocs.io/en/master/dev/development_guidelines.html#schema-changes).
    - If you change the schema, meadow, or guest language interfaces, please follow the scheme [for internal versioning in the developer guidelines](https://ensembl-hive.readthedocs.io/en/master/dev/development_guidelines.html#internal-versioning).
    - The PR must not fail unit testing.

## Use case

This makes @carlacummins peekJob.pl script available for users running version/2.5 of eHive

## Description

The commits from PR #114 have been cherry-picked onto version/2.5. I updated the copyrights to 2020 - they were updated during the annual copyright update for master, and would have been updated in version/2.5 had they been merged before the end of 2019

## Possible Drawbacks

**NOTE!** after this PR is merged, there needs to be a merge to master using `git merge -s ours version/2.5`. This will keep the merge history clear and facilitate further cascade merges.

This is a variance from our usual policy of only incorporating bugfixes into released versions of eHive. As this only adds functionality, is widely requested, and introduces no backwards compatibility issues, we are making an exception to add this.

## Testing

Tests included in the cherry-pick

_If so, do the tests pass/fail?_
pass

_Have you run the entire test suite and no regression was detected?_
no regression

### CODE REVIEW

**Comment by ens-bwalts** [comment]
To re-emphasize, after merging this PR, there must be a cascade-merge to master with `git merge -s ours version/2.5` to keep a clean commit history and allow further cascade merges.

File: modules/Bio/EnsEMBL/Hive/Utils/Test.pm [hunk 1]
```diff
@@ -46,7 +46,7 @@ use Bio::EnsEMBL::Hive::Scripts::StandaloneJob;
 our @ISA         = qw(Exporter);
 our @EXPORT      = ();
 our %EXPORT_TAGS = ();
-our @EXPORT_OK   = qw( standaloneJob init_pipeline runWorker beekeeper generate_graph visualize_jobs db_cmd seed_pipeline get_test_urls get_test_url_or_die run_sql_on_db load_sql_in_db make_new_db_from_sqls make_hive_db safe_drop_database all_source_files);
+our @EXPORT_OK   = qw( standaloneJob init_pipeline runWorker beekeeper generate_graph visualize_jobs db_cmd seed_pipeline tweak_pipeline peekJob get_test_urls get_test_url_or_die run_sql_on_db load_sql_in_db make_new_db_from_sqls make_hive_db safe_drop_database all_source_files);
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Hive/Utils/Test.pm:49]: There is no `tweak_pipeline` in the `version/2.5` version of this file
```suggestion
our @EXPORT_OK   = qw( standaloneJob init_pipeline runWorker beekeeper generate_graph visualize_jobs db_cmd seed_pipeline peekJob get_test_urls get_test_url_or_die run_sql_on_db load_sql_in_db make_new_db_from_sqls make_hive_db safe_drop_database all_source_files);
```

**Review by muffato — CHANGES_REQUESTED** [review]
Just a minor change

File: modules/Bio/EnsEMBL/Hive/Utils/Test.pm (hunk 1 above)
Comment (ens-bwalts) [reply modules/Bio/EnsEMBL/Hive/Utils/Test.pm]: You are right, removed.

**Review by muffato — APPROVED** [review]

**Review by mira13 — APPROVED** [review]
Looks fine, applying changes to the version 2.5

**Pull request merged**